src/main/io/osd.c

```diff
@@ -1829,6 +1830,18 @@ static void osdShowStats(uint16_t endBatteryVoltage)
     }
 #endif
 
+#ifdef USE_GYRO_DATA_ANALYSE
```

Comment:
SITL failing.

Change to:
`#if (defined(USE_GYRO_DATA_ANALYSE) && !defined(SIMULATOR_BUILD))`